src/Compilers/CSharp/Test/Semantic/Semantics/ExtensionTypeTests.cs

```diff
@@ -75,14 +76,12 @@ public class ExtensionTypeTests : CompilingTestBase
 
         foreach (var baseExtension in namedType.BaseExtensionsNoUseSiteDiagnostics)
         {
-            if (baseExtension.IsExtension)
-            {
-                VerifyExtension<TypeSymbol>(baseExtension, isExplicit: null);
-            }
-            else
-            {
-                Assert.True(baseExtension.IsErrorType());
-            }
+            checkBaseExtension(baseExtension);
```

Comment:
>checkBaseExtension(baseExtension); [](http://example.com/codeflow?start=12&length=34)

It feels redundant to do this since the loop below does that as well.